src/Services/Authentication/Authentication.BusinessLogic/Services/Implementations/AuthService.cs

```diff
@@ -30,17 +34,20 @@ public async Task<GeneralResponseDto> RegisterAsync(UserRequestDto registrationR
 
     public async Task<LoginResponseDto> LoginAsync(UserRequestDto loginRequestDto, CancellationToken cancellationToken)
     {
+        await _validator.ValidateAndThrowAsync(loginRequestDto, cancellationToken);
         var user = await _userRepository.GetByEmailAsync(loginRequestDto.Email, cancellationToken);
             
         if (user is null)
         {
+            _logger.LogError($"User with email = {loginRequestDto.Email} was not found");
             throw new LoginException(ExceptionMessages.LoginFailed);
         }
 
         var isValid = await _userRepository.CheckPasswordAsync(user, loginRequestDto.Password);
 
         if (isValid == false)
         {
+            _logger.LogError("Username or password is incorrect");
```

Comment:
It is better to write more informative logs so that it is clear. 
Let's say specifying information about the account we are trying to log in to, but without a password